# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437852319 14400
#      Sat Jul 25 15:25:19 2015 -0400
# Node ID 4e52267743d736edb94c256893b2d01c78ad2ffc
# Parent  2f2e8b4996cb6fb111ac6dbfaa0e11009093d8e2
maint: Delete obsolete find-docstring-files.sh build script

* doc/interpreter/find-docstring-files.sh: Delete.
* doc/module.mk (EXTRA_DIST): Remove doc/interpreter/find-docstring-files.sh
from the list.

diff --git a/doc/interpreter/find-docstring-files.sh b/doc/interpreter/find-docstring-files.sh
deleted file mode 100755
--- a/doc/interpreter/find-docstring-files.sh
+++ /dev/null
@@ -1,27 +0,0 @@
-#! /bin/sh
-
-if [ $# -ne 1 ]; then
-  echo "usage: find-docstring-files TOP-SRCDIR" 1>&2
-  exit 1
-fi
-
-## Look in the build tree then the source tree for the
-## DOCSTRINGS files.  Fail if neither exists.
-
-if [ -f "scripts/DOCSTRINGS" ]; then
-  echo "scripts/DOCSTRINGS"
-elif [ -f "$1/scripts/DOCSTRINGS" ]; then
-  echo "$1/scripts/DOCSTRINGS"
-else
-  echo "find-docstring-files: scripts/DOCSTRINGS file is missing!" 1>&2
-  exit 1
-fi
-
-if [ -f "libinterp/DOCSTRINGS" ]; then
-  echo "libinterp/DOCSTRINGS"
-elif [ -f "$1/libinterp/DOCSTRINGS" ]; then
-  echo "$1/libinterp/DOCSTRINGS"
-else
-  echo "find-docstring-files: libinterp/DOCSTRINGS file is missing!" 1>&2
-  exit 1
-fi
diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -343,17 +343,16 @@ INSTALL.OCTAVE: doc/interpreter/install.
 	$(MAKEINFO) -D INSTALLONLY -I $(srcdir)/doc/interpreter \
 	  --no-validate --no-headers --no-split --output $@-t $< && \
 	mv $@-t $@
 
 doc_EXTRA_DIST += \
   doc/interpreter/config-images.sh \
   doc/interpreter/contributors.in \
   doc/interpreter/doc-cache \
-  doc/interpreter/find-docstring-files.sh \
   doc/interpreter/genpropdoc.m \
   doc/interpreter/graphics_properties.mk \
   doc/interpreter/images \
   doc/interpreter/images.awk \
   doc/interpreter/images.mk \
   doc/interpreter/macros.texi \
   doc/interpreter/mk_doc_cache.m \
   doc/interpreter/mkcontrib.awk \
